src/html/assets/themes/fabrizio1.0/templates/partials/header-secondary.php

```diff
@@ -17,10 +17,6 @@
                 <li><a href="#" class="triggerShare"><i class="icon-share-alt triggerShare"></i> Share this album</a></li>
               <?php } ?>
             <?php } ?>
-          <?php } elseif($this->utility->isActiveTab('upload')) {?>
```

Comment:
Yes, if we have applications available on the store., it makes sense to keep theses links.